extensions/container-image/container-image-s2i/deployment/src/main/java/io/quarkus/container/image/s2i/deployment/S2iConfig.java

```diff
@@ -15,7 +15,7 @@ public class S2iConfig {
 
     public static final String DEFAULT_BASE_JVM_JDK11_IMAGE = "registry.access.redhat.com/ubi8/openjdk-11:1.14";
     public static final String DEFAULT_BASE_JVM_JDK17_IMAGE = "registry.access.redhat.com/ubi8/openjdk-17:1.14";
```

Comment:
Shouldn't this be `openjdk-17:1.15` like in 2 other cases?

Reply:
It should and that's concerning as it looks like the previous backport didn't change them. I need to have a closer look.

Reply:
It looks like we are using a floating tag for these 2 in main so that's why they weren't updated in main and thus not backported.
I'll go fix that in an additional commit.